web/src/main/webapp/WEB-INF/classes/org/fao/geonet/api/Messages.properties

```
@@ -77,6 +81,21 @@ register_email_message=Dear User,\n\
   \n\
   Yours sincerely,\n\
   The %s team.
+register_email_group_message=Dear User,\n\
+  Your registration at %s was successful.\n\
+  Your account is: \n\
+  * username: %s\n\
+  * password: %s\n\
+  * profile: %s\n\
+  \n\
+  You've told us that you want to be %s in group %s, you will be contacted soon.\n\
```

Not sure about the duplication with core/src/test/resources/org/fao/geonet/api/Messages.properties  above; I assume this is the one to fix.

```suggestion
  We have sent your request for %s to the group %s administrator. You will be contacted shortly.\n\
```